pkg/machinery/constants/constants.go

```diff
@@ -163,11 +163,14 @@ const (
 	KubernetesEtcdListenClientPort = "2379"
 
 	// KubernetesAdminCertCommonName defines CN property of Kubernetes admin certificate.
-	KubernetesAdminCertCommonName = "apiserver-kubelet-client"
+	KubernetesAdminCertCommonName = "admin"
```

Comment:
Does this need to be backported?

Reply:
yep, it's in the `Backports 0.11` project!